episodes/06-blurring.md

```diff
@@ -327,7 +327,143 @@ fig, ax = plt.subplots()
 plt.imshow(blurred)
 ```
 
-![](fig/gaussian-blurred.png){alt='Original image'}
+![](fig/gaussian-blurred.png){alt='Blurred image'}
+
+
+## Visualizing Blurring
```

Comment:
```suggestion
## Visualising Blurring
```

UK English spelling to conform to [The Carpentries Style Guide](https://docs.carpentries.org/topic_folders/communications/resources/style-guide.html#the-carpentries-style-guide).